dashboard/app/mailers/pd/teacher_application_mailer.rb

```diff
@@ -15,6 +15,8 @@ def principal_approval_request(teacher_application)
     @teacher_name = teacher_application.teacher_name
     @program_name = teacher_application.program_name
     @program_url = teacher_application.program_url
+    @due_date = @program_name == Pd::Workshop::COURSE_CSP ? 'May 5, 2017' : 'April 17, 2017'
```

Comment:
No, it's [always](https://github.com/code-dot-org/code-dot-org/blob/f8ec05fd0f70c8dce246eaa7fe636851e1829198/dashboard/app/models/pd/teacher_application.rb#L77-L77) csd or csp. Will update